portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -116,5 +125,100 @@ private int getCommentLimit(HttpServletRequest request, int total) {
 
     return commentHistory;
   }
+
+  /**
+   * @return the Comment Data from Datastore
+   */ 
+  private Entity getCommentData() {
+      handleCommentData(); 
```

Comment:
Rather than have two similar utility functions, it would be nice to put a common function for `handleCommentData` here and `handleRetrievedData` above in some kind of utils file and import it.